doc/source/conf.py

```diff
@@ -237,14 +237,14 @@
 
 html_theme_options = {
     "external_links": [],
-    "footer_items": ["pandas_footer", "sphinx-version"],
+    "footer_start": ["pandas_footer", "sphinx-version"],
     "github_url": "https://github.com/pandas-dev/pandas",
     "twitter_url": "https://twitter.com/pandas_dev",
-    "google_analytics_id": "UA-27880019-2",
+    "analytics": {"google_analytics_id": "UA-27880019-2"},
     "logo": {"image_dark": "https://pandas.pydata.org/static/img/pandas_white.svg"},
     "navbar_end": ["version-switcher", "theme-switcher", "navbar-icon-links"],
     "switcher": {
-        "json_url": "/versions.json",
+        "json_url": "versions.json",
```

Comment:
Seems like the dropdown is broken with this change: https://pandas.pydata.org/preview/53029/docs/

What was the problem that made you change this?

Reply:
Seems like in 0.13 this is the code that reads the json_url:

https://github.com/pydata/pydata-sphinx-theme/blob/f15ecfed59a2a5096c05496a3d817fef4ef9a0af/src/pydata_sphinx_theme/__init__.py#L105

With the `/version.json` for example in https://github.com/pandas-dev/pandas/actions/runs/4863923942/jobs/8672280782, I would get

```
2023-05-02T17:22:30.6605921Z WARNING: The version switcher "/versions.json" file cannot be read due to the following error:
2023-05-02T17:22:30.6606819Z FileNotFoundError(2, 'No such file or directory')
```

Reply:
Ah, I see. Not great, but then I think we need this:

```suggestion
        "json_url": "https://pandas.pydata.org/versions.json",
```

Reply:
Great this worked thanks!